src/Ng/Program.cs

```diff
@@ -27,22 +27,22 @@ public static void Main(string[] args)
                 Debugger.Launch();
             }
 
-            // Get arguments
-            var arguments = CommandHelpers.GetArguments(args, 0);
-
-            // Configure ApplicationInsights
-            ApplicationInsights.Initialize(CommandHelpers.GetApplicationInsightsInstrumentationKey(arguments));
-
             // Create an ILoggerFactory
             var loggerConfiguration = LoggingSetup.CreateDefaultLoggerConfiguration(withConsoleLogger: true);
             var loggerFactory = LoggingSetup.CreateLoggerFactory(loggerConfiguration);
 
             // Create a logger that is scoped to this class (only)
             _logger = loggerFactory.CreateLogger<Program>();
```

Comment:
The reason this wasn't done was because the catch statements of the try block below both try to log error using this logger. It looks like we might have to change that if this require that AI initialize before it works correctly.

---
In reply to: [71253697](https://github.com/NuGet/NuGet.Services.Metadata/pull/77#discussion_r71253697) [](ancestors = 71253697)